app/javascript/modules/listingDetailsAside/ListingDetailsApply.tsx

```diff
@@ -1,85 +1,183 @@
+} from "../../util/listingUtil"
+import { localizedFormat, renderInlineWithInnerHTML } from "../../util/languageUtil"
 
 export interface ListingDetailsApplyProps {
   listing: RailsListing
 }
 
-/*
-  TODO: this component will also include
-  How to Apply
-  Need Help
-*/
 export const ListingDetailsApply = ({ listing }: ListingDetailsApplyProps) => {
-  const availableDescription = <p>{t("listings.availableUnitsAndWaitlistDescription")}</p>
+  const [paperApplicationsOpen, setPaperApplicationsOpen] = useState(false)
 
-  const availableRows = [
-    {
-      amount: listing.Units_Available,
-      text: t("listings.availableUnits"),
-      emphasized: true,
-    },
-    {
-      amount: listing.Total_waitlist_openings,
-      text: t("listings.openWaitlistSlots"),
-      emphasized: true,
-    },
-  ]
+  if (!isOpen(listing)) return null
 
-  const unavailableDescription = (
+  const isListingRental = isRental(listing)
+
+  const acceptingPaperApps = acceptingPaperApplications(listing)
+
+  const ordinalHeader = (ordinal: number, title: string) => {
+    return (
+      <Heading priority={4} className={"text-gray-900 text-lg -mt-2 mb-3"}>
+        <span className={"text-blue-600 mr-2"}>{ordinal}</span>
+        {title}
+      </Heading>
+    )
+  }
+
+  const howToApplyBlock = (
+    <SidebarBlock title={t("listings.apply.howToApply")}>
+      {!isListingRental && (
+        <>
+          <p className={"mb-4"}>
+            {renderInlineWithInnerHTML(
+              t("listings.apply.fulfillEligibilityRequirements", {
+                url: "https://sfmohcd.org/homebuyer-program-eligibility",
+              })
+            )}
+          </p>
+          <p className={"mb-4"}>{t("listings.apply.eligibilityRequirementDescription")}</p>
+        </>
+      )}
+      <LinkButton
+        styleType={AppearanceStyleType.primary}
+        className={"w-full"}
+        transition={true}
+        href={`listings/${listing.listingID}/apply-welcome/intro`}
+      >
+        {t("label.applyOnline")}
+      </LinkButton>
+      {process.env.COVID_UPDATE && (
+        <div className={"mt-4"}>
+          <Heading priority={4} className={"text-base text-gray-800 font-sans"}>
+            {t("listings.apply.covidUpdate")}
+          </Heading>
+          <div className={"text-gray-700 text-base mt-2"}>
+            {t("listings.apply.covidUpdateInfo")}
+          </div>
+        </div>
+      )}
+      {acceptingPaperApps && (
+        <>
+          <OrDivider bgColor={"white"} />
+          {ordinalHeader(1, t("listings.apply.getAPaperApplication"))}
+          {t("listings.apply.paperApplicationsMustBeMailed")}
+          <Button
+            className={"w-full mt-4"}
+            transition={true}
+            icon={"arrowDown"}
+            iconPlacement={"right"}
+            onClick={() => setPaperApplicationsOpen(!paperApplicationsOpen)}
+          >
+            {t("label.downloadApplication", {
+              lang: "",
+            })}
```

Comment:
could we remove this lang param from here and the text props?

Reply:
Can do! Needed to add a new string